catalog/imaging/imaging-plugin-nitf/src/main/java/org/codice/alliance/plugin/nitf/NitfPostIngestPlugin.java

```diff
@@ -201,34 +245,18 @@ private void process(ContentItem contentItem, List<ContentItem> contentItems) {
 
         if (storeOriginalImage) {
           ContentItem originalImageContentItem =
-              createOriginalImage(
-                  contentItem.getId(), renderImageUsingOriginalDataModel(contentItem), metacard);
+              createOriginalImage(contentItem.getId(), renderedImage, metacard);
 
           contentItems.add(originalImageContentItem);
         }
       }
-    } catch (IOException | ParseException | NitfFormatException | RuntimeException e) {
+    } catch (IOException | NitfFormatException | RuntimeException e) {
       LOGGER.debug(e.getMessage(), e);
```

Comment:
This debug message could be update to something more meaningful, like `LOGGER.debug("Error creating and storing thumbnail/overview/original: {}", e.getMessage(), e);`. Also, is it possible that an exception would be thrown during process of only one of the derived images would fail but we'd still want to continue to see if we can create the other images?

Reply:
Updated logging. Since they all use the same rendered image it is all or nothing right now.